helm/azure-operator-resource-chart/templates/cluster.yaml

```diff
@@ -25,6 +25,7 @@ spec:
       cidr: "{{ .Values.azure.cidr }}"
       masterSubnetCIDR: "{{ .Values.azure.masterSubnetCIDR }}"
       workerSubnetCIDR: "{{ .Values.azure.workerSubnetCIDR }}"
+      vpnSubnetCidr: "{{ .Values.azure.vpnSubnetCidr }}"
```

Comment:
I think we should just remove `examples` directory and `helm/azure-operator-resource-chart` directory.

Reply:
Some cleaning would be needed there, let's do that in a different PR.